src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourcePagingTest.kt

```diff
@@ -131,4 +132,56 @@ class AuditResourcePagingTest : IntegrationTest() {
       .jsonPath("$.last").isEqualTo(false)
       .jsonPath("$.content[0].who").isEqualTo("bobby.beans")
   }
+
+  @Suppress("ClassName")
+  @Nested
+  inner class filteredAuditEvents {
+
+    @Test
+    fun `filter audit events by what`() {
+      webTestClient.get().uri("/audit/paged?page=0&size=3&what=OFFENDER_DELETED")
+        .headers(setAuthorisation(roles = listOf("ROLE_AUDIT")))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("$.content.length()").isEqualTo(2)
+        .jsonPath("$.size").isEqualTo(3)
+        .jsonPath("$.totalElements").value<Int> { Assertions.assertThat(it).isEqualTo(2) }
```

Comment:
I think for `isEqualTo` you don't know to extract the value to assert... you only need the extraction if you need a fancy AssertJ assertion like `contains` or `greaterThan`.

Reply:
yes, I will change this in my next push back. cheers